source/documentation/data-docs/data-faqs/index.md

```diff
@@ -0,0 +1,43 @@
+# Data FAQs
+
+## Where do I find out what data is already on the Platform?
+
+The data engineering team maintain a number of databases on the Platform (curated databases). The best way to find out about these is using the [data discovery tool](https://data-discovery-tool.apps.alpha.mojanalytics.xyz). To access the tool, you must sign in using an email link. Many users will have had their justice emails automatically added to the allow list. If you enter your email and are denied access, you can request to be added in `#ask-data-engineering` in Data and Analysis slack. The tool can be updated by anyone, so if you find something out about the data that isn't already documented, please do add to it!
+
+In addition to this users can create their own S3 buckets which may have data useful to other teams, you may have to ask around to see if there is an existing dataset that may suit your needs.
+
+## How do I gain access to existing data?
+
+Access to curated databases is granted via the [database access repository](https://github.com/moj-analytical-services/data-engineering-database-access). Have a read through the guidance on there. If you are finding the process a little tricky, please ask for help in `#ask-data-engineering`. A data engineer will happily guide you through things. 
+
+If you are looking for access to a user created bucket, then the admin of that bucket should be able to grant you access. If you don't know who the admin is, or they are not able to grant you access, then ask in the `#analytical-platform` slack channel.
+
+## Where should I store my own data?
+
+Data should be stored in an s3 bucket. You can create a new s3 bucket in the control panel. Data can be uploaded manually via the AWS console (which can be accessed through the control panel) or you can write it from RStudio or JupyterLab.
```

Comment:
Do you need to add something about data migration forms? Or like please see this section to check you are ok to upload this data to the platform

Reply:
Yes agreed - I'm working with data privacy to add some wording but for now just say something like 'you should ensure you have permission from the Information Asset Owner to move the data into the Analytical Platform and have filled out a data movements form'

Question that's just occurred to me - do people only need data movement forms for personal data, or all data?